### DISCUSSION TITLE
Allow Deno usage in addition to Node

### DISCUSSION THREAD

**Discussion by backspaces** [11ty/eleventy#2953] [Uncategorized]
Deno is really lovely to use, being close to the browser's Javascript, having simple file access, uses es5 modules, and is fast to grok.

It is also quickly adding Node extensions so that porting Node apps to Deno is becoming easer.

Would it be possible for 11ty to allow Deno use where now Node is required?

**Comment by manicExpressive** [comment]
I'm not sure exactly what's different between when you asked this and now, but Deno works pretty well with 11ty at this point - I've been able to run something like:

> `deno npm:@11ty/eleventy --serve`

in a few small 11ty projects without making any changes at all and it has done a great job.
